src/utils/helpers.ts

```diff
@@ -0,0 +1,20 @@
+import { Parameter } from 'src/Interfaces/Parameter';
+
+export function parameterArrayToMatrix(
```

Comment:
This shouldn't just be specific to `Parameter`. This would be a generic function, which can take an array of any values (so `Array<any>` and return you a matrix based on the number of items you want per row). So you can name your parameters, `arrayToConvert` and `itemsPerRow`